### PR TITLE
Feature/pok 77 types text

### PR DESCRIPTION
**Pull request by adCBueno**
Text changed to uppercase, let me know if it should be changed to capitalized but I think this way looks better

Captura de pantalla 2023-04-11 a la(s) 11 05 29

Captura de pantalla 2023-04-11 a la(s) 11 05 48

### CODE REVIEW

File: PokeAppAndroid/Adapters/PokemonTypeAdapter.cs [hunk 1]
```diff
@@ -27,8 +27,10 @@ public override void OnBindViewHolder(RecyclerView.ViewHolder holder, int positi
         {
             TypesViewHolder typesViewHolder = holder as TypesViewHolder;
             string item = types[position];
-            typesViewHolder.TvTypeName.Text = item;
+            typesViewHolder.TvTypeName.Text = item.ToUpper();
```
Comment (rodrigoschar) [inline PokeAppAndroid/Adapters/PokemonTypeAdapter.cs]: Shouldn't only the first letter be capitalized instead of the whole word?

**Review by YawarValeriano — CHANGES_REQUESTED** [review]
There are also types in detail view
Screenshot 2023-04-11 at 12 14 29

File: PokeAppAndroid/Adapters/PokemonTypeAdapter.cs (hunk 1 above)
Comment (adCBueno) [reply PokeAppAndroid/Adapters/PokemonTypeAdapter.cs]: Thought it looked good but no problem, corrected :)

File: PokeAppiOS/Views/Cells/TypeCollectionViewCell.cs [hunk 2]
```diff
@@ -21,7 +20,8 @@ public TypeCollectionViewCell (IntPtr handle) : base (handle)
 		public void UpdateCell()
 		{
             if (TypeName == null) return;
-            typeLabel.Text = TypeName;
+            typeLabel.Text = TypeName.ToUpper();
```
[the remark this answers is not in the document]
Comment (adCBueno) [reply PokeAppiOS/Views/Cells/TypeCollectionViewCell.cs]: Corrected, I don't have the detail of Android yet tho

File: PokeAppAndroid/Adapters/PokemonTypeAdapter.cs [hunk 3]
```diff
@@ -27,8 +27,10 @@ public override void OnBindViewHolder(RecyclerView.ViewHolder holder, int positi
...
             typesViewHolder.TvTypeName.SetDrawableBackgroundForType(item);
+            typesViewHolder.TvTypeName.SetTextSize(Android.Util.ComplexUnitType.Sp, 10);
```
[the remark this answers is not in the document]
Comment (adCBueno) [reply PokeAppAndroid/Adapters/PokemonTypeAdapter.cs]: When changed to uppercase it was covering the pokemon image, but corrected

**Review by rodrigoschar — APPROVED** [review]

**Review by jimibilibob — APPROVED** [review]

**Review by YawarValeriano — APPROVED** [review]

**Pull request merged**